spyder/utils/conda.py

```diff
@@ -114,12 +112,23 @@ def get_conda_env_path(pyexec, quote=False):
 
 def find_conda():
     """Find conda executable."""
-    # First try the environment variables
-    conda = os.environ.get('CONDA_EXE') or os.environ.get('MAMBA_EXE')
+    conda = None
+
+    # First try Spyder's conda executable
+    if is_conda_based_app():
+        # TODO: Change to CONDA_EXE when
+        # conda-forge/conda-standalone-feedstock#45 is resolved
+        conda = osp.dirname(os.environ.get('CONDA_PYTHON_EXE')) + '/bin/mamba'
```

Comment:
Two questions about this:

1. Are we going to package `mamba` or `micromamba` in our installers? This code implies that we're going to use `mamba`, but why not micromamba since it's a single binary?
2. Does this work on Windows? I think executables are put in `path\to\env\Library\bin` there.

Reply:
Yes, we should have a deeper discussion about that.

Originally, I thought that `conda.exe` would be a direct replacement for `micromamba`. But per @jaimergp's comments, that is not the case (at least not yet). Currently, the conda-based installer includes `mamba` in the base environment, so it was already there and `micromamba` would be another element to deal with. However, we could possibly omit conda and mamba from the base environment and include micromamba instead.

Reply:
Take into account that micromamba ships its own implementation of `menuinst 1.x`, in case that's relevant for your shortcuts.

Reply:
@jaimergp, thanks for clarifying. I don't think it will matter for us, since `conda.exe` is handling the shortcuts for us at install time. The other executable ('mamba' or 'micromamba') included with the base environment will be used by the Spyder application at runtime to do various things.

If `conda.exe` ever becomes an effective equivalent to `micromamba`, then we could just use that and not worry about including any other flavor.

Reply:
> the conda-based installer includes mamba in the base environment

Ok, I didn't know that. Then I think it's fine to use `mamba` here.

> Take into account that micromamba ships its own implementation of menuinst 1.x, in case that's relevant for your shortcuts.

Thanks for letting us know about that @jaimergp.

Reply:
> So it seems that conda cannot be omitted in favor of micromamba. Is this still the case, or is the documentation outdated?

I guess we can make it work, but for now there are strict checks for the `conda` package itself. It's not needed for the solve but the code operates under the assumption that if you want a separate environment it's because you are going to use `conda` on it, otherwise you could just deploy to a conda-less `base`.

Happy to revisit that decision, though!

Reply:
> Happy to revisit that decision, though!

No need, I just wanted to clarify what could or could not be done. Thanks.
Given what I now understand, I think the best path forward is to just include mamba in the base environment, use this for Spyder's runtime conda executable, and then there is no need for micromamba. There are also some minor differences between micromamba and mamba with regard to their `info` command that makes mamba more preferable anyway.

Reply:
If you already have a setup that supports `mamba`, very little I'd say. We have some tighter integration with the classic solver, but it's no big deal. The main advantage is not having to have code that checks for `mamba` or `conda` executables; it's just `conda` with some configuration on `.condarc`.